stacker_blueprints/conveyor/__init__.py

```diff
@@ -0,0 +1,482 @@
+ELB_SG_NAME = 'ConveyorELBSecurityGroup'
+LAUNCH_CONFIGURATION = 'ConveyorLaunchConfiguration'
+LOAD_BALANCER = 'LoadBalancer'
+LOG_GROUP = 'LogGroup'
+QUEUE = 'Queue'
+
+
+class Conveyor(Blueprint):
+
+    LOCAL_PARAMETERS = {
+        'GitHubToken': {
+            'type': str,
+            'description': 'GitHub API token to use when creating commit statuses.',
+        },
+        'SshIdRsa': {
+            'type': str,
+            'description': 'SSH id_rsa for the bot github user.',
+        },
+        'SshIdRsaPub': {
+            'type': str,
+            'description': 'SSH id_rsa.pub for the bot github user.',
+        },
+        'GitHubSecret': {
+            'type': str,
+            'description': 'The shared secret that GitHub uses to sign webhook payloads.',
+            'default': '',
+        },
+        'SlackToken': {
+            'type': str,
+            'description': 'Secret shared with Slack to verify slash command webhooks.',
+            'default': '',
+        },
+        'DataDogApiKey': {
+            'type': str,
+            'description': 'If provided, metrics will be collected and sent to datadog.',
+            'default': '',
+        },
+        'DockerConfig': {
+            'type': str,
+            'description': 'Contents of a .docker/config.json file',
+            'default': '',
+        },
+    }
+
+    PARAMETERS = {
+        'VpcId': {
+            'type': 'AWS::EC2::VPC::Id',
+            'description': 'ID of the VPC to launch Conveyor in.',
+        },
+        'DefaultSG': {
+            'type': 'AWS::EC2::SecurityGroup::Id',
+            'description': 'Top level security group.',
+        },
+        'PrivateSubnets': {
+            'type': 'List<AWS::EC2::Subnet::Id>',
+            'description': 'Subnets to deploy private instances in.',
+        },
+        'PublicSubnets': {
+            'type': 'List<AWS::EC2::Subnet::Id>',
+            'description': 'Subnets to deploy public (elb) instances in.',
+        },
+        'AvailabilityZones': {
+            'type': 'List<AWS::EC2::AvailabilityZone::Name>',
+            'description': 'Comma delimited list of availability zones. MAX 2',
+        },
+        'SshKeyName': {
+            'type': 'AWS::EC2::KeyPair::KeyName',
+            'description': 'The name of the key pair to use to allow SSH access.',
+        },
+        'TrustedNetwork': {
+            'type': 'String',
+            'description': 'CIDR block allowed to connect to Conveyor ELB.'
+        },
+        'SSLCertificateName': {
+            'type': 'String',
+            'description': (
+                'The name of the SSL certificate to attach to the load'
+                ' balancer.  Note: If this is set, non-HTTPS access is disabled.'
+            ),
+            'default': '',
+        },
+        'ExternalDomain': {
+            'type': 'String',
+            'description': 'Base domain for the stack.',
+            'default': '',
+        },
+        'Subdomain': {
```

Comment:
A lot of the same comments as the Drone blueprint - this seems like the host part of the hostname, not a subdomain.